### PR TITLE
nspawn: Use "ctrl-" rather than "^" in info msg

### PR DESCRIPTION
**Pull request by mwleeds**
Maybe most people know that "^]" means "Ctrl + ]" but for those that don't, this should be more clear.

### CODE REVIEW

File: src/nspawn/nspawn.c [hunk 1]
```diff
@@ -5798,7 +5798,7 @@ static int run(int argc, char *argv[]) {
                 arg_quiet = true;
 
         if (!arg_quiet)
-                log_info("Spawning container %s on %s.\nPress ^] three times within 1s to kill container.",
+                log_info("Spawning container %s on %s.\nPress ctrl-] three times within 1s to kill container.",
```
Comment (yuwata) [inline src/nspawn/nspawn.c]: Maybe `Ctrl` (capitalized) is better? Though, not a strong opinion.

**Review by yuwata — APPROVED** [review]

File: src/nspawn/nspawn.c (hunk 1 above)
Comment (brauner) [reply src/nspawn/nspawn.c]: Agreed but equally not fuzzed.

**Review by brauner — APPROVED** [review]

**Comment by poettering** [comment]
yeah, please uppercase the `C`

File: src/nspawn/nspawn.c (hunk 1 above)
Comment (mwleeds) [reply src/nspawn/nspawn.c]: Okay, pushed that change

**Review by brauner — APPROVED** [review]

**Pull request merged**